charts/multicluster-controlplane/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for multicluster-controlplane.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/open-cluster-management/multicluster-controlplane:latest
+imagePullPolicy: IfNotPresent
+
+replicas: 1
+
+configfile: "ocmconfig.yaml"
+serviceType: ClusterIP
```

Comment:
I think we may have three types to support: ocproute, lb and nodeport, for nodeport, and we may use independent config to represent them, maybe

```
ocproute:
   enabled: true

loadbalance:
   enabled: false
   baseDomain: ""  #e,g, *.aws.xxx

nodeport:
   enabled: false
   nodeport: 30443
```